app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -67,6 +67,8 @@ export default Controller.extend(bufferedProperty("model"), {
   username_filters: null,
   filter: null,
   quoteState: null,
+  _retryRateLimited: false,
+  _newPostsInStream: [],
```

Comment:
```suggestion
  _newPostsInStream: null,
```

and:

```
init() {
  this._super(...arguments);
  this.set("_newPostsInStream", []);
```

Or if you don't need _newPostsInStream to be computed, you could just remove all of this and do:

```
init() {
  this._super(...arguments);
  this. _newPostsInStream = [];
}
```

If you want to know more about this: https://dockyard.com/blog/2015/09/18/ember-best-practices-avoid-leaking-state-into-factories